packages/apollo-server-express/src/ApolloServer.test.ts

```diff
@@ -363,6 +363,11 @@ describe('apollo-server-express', () => {
     });
     describe('file uploads', () => {
       it('enabled uploads', async () => {
+        //XXX This is currently a failing test for node 10
```

Comment:
Would you like to pair on tracking this down? Node 10 is released so it seems like we should make this work.